### PR TITLE
Changelog 24.11.0

### PR DESCRIPTION
**Pull request by beeankha**
## Description

✂️ snip snip ✂️ the making of a new release.

Xref https://github.com/conda/conda-build/issues/5524

### CODE REVIEW

File: CHANGELOG.md
Comment (beeankha) [inline CHANGELOG.md:21]:
```suggestion
* Fix regex for Jinja2 `set` / `for` statements to be more specific. (#5514)
* Fix `ruamel.yaml` usage to use supported APIs. (#5517)
* Fix bug in docs build by pinning `conda-sphinx-theme` version to 0.2.2. (#5518)
* Fix bug variant variables were not defined for the first parsing pass of a recipe. (#5528)
* Fix a bug where variants were incorrectly found as being used when they matched a leading substring of
  another variant. (#5535)
* Fix a bug where variants were not found when variables were used in `pin_*` statements. (#5535)
```

File: CHANGELOG.md
Comment (beeankha) [inline CHANGELOG.md:7]:
```suggestion
* Introduce `--package-format` as command line argument. (#4890 via #5209)
```

File: .authors.yml
Comment (beeankha) [inline .authors.yml:1254]: Not sure why the number of commits got lower here... 🤔

File: CHANGELOG.md
Comment (beeankha) [inline CHANGELOG.md:7]:
```suggestion
* Introduce `--package-format` as a command line argument. (#4890 via #5209)
```

File: CHANGELOG.md
Comment (beeankha) [inline CHANGELOG.md:11]:
```suggestion
* Add  support for [CEP-17](https://github.com/conda/ceps/blob/main/cep-0017.md) that allows specifying the location of the site-packages directory with the `python_site_packages_path` build option for any packages named `python`. (#5502)
```

File: .authors.yml
Comment (beckermr) [reply .authors.yml:1254]: I'd bump it right back up!

**Review by beckermr — APPROVED** [review]
LGTM! We may want to wait until Monday to merge to avoid breaking things over the weekend.

**Comment by beeankha** [comment]
@beckermr thank you for the review! Agreed, I'd like to wait to merge this (+ tag the 24.11.0 release) until next week.

File: CHANGELOG.md
Comment (jaimergp) [inline CHANGELOG.md:17]: Does this belong to `# Docs` better?

**Review by jaimergp — APPROVED** [review]

**Review by jezdez — APPROVED** [review]

File: CHANGELOG.md
Comment (jezdez) [reply CHANGELOG.md]: There is a bug in rever that miscounts things, was fixed in https://github.com/regro/rever/pull/266 but that was never released, so you likely used a version that miscounts contributions basically

Comment (jezdez) [reply CHANGELOG.md]: You can redo the run if you want with that new version

File: .authors.yml
Comment (kenodegard) [reply .authors.yml:1254]: The commit counts are weird for whomever last did the release due to our commit squashing policy

**Pull request merged**
